### ISSUE TITLE
[WASM] Drag Icon do not move in the iframe html element.

### ISSUE THREAD

**Issue by TopProgrammer77** [unoplatform/uno#14721]
### Current behavior

In our wasm app, `iframe` is used instead of `WebView2`. This `iframe` element allow Drag/Drop by `AllowDrop=true`.
But when dragging text or file to inside of iframe, the icon tooltip of drag doesn't move.
Also when dropping it in the iframe, the tooltip doesn't disappear.

-screenshot
![image](https://github.com/unoplatform/uno/assets/122166904/4cffaaa3-9731-4c1b-ab2a-cd98af6b6c30)

-video 
![ice_video_20231206-015855_edit_0](https://github.com/unoplatform/uno/assets/122166904/9ee2883e-12bd-468a-9dab-6d0d6beddfb8)

- code

[1.zip](https://github.com/unoplatform/uno/files/13577691/1.zip)

### Works on UWP/WinUI

None

**Comment by DierkDroth** [comment]
@jeromelaban this actually is a weird one. We're not even sure how this could be addressed properly:
- is the problem that the mouse-button-released event just not propagated as it should (hence, the tooltip/icon is vanishing) OR
- should UNO **not** show a tooltip in the first place (at least not on Windows) since MS Windows takes care anyway, and any tooltip/icon actually would be redundant

**Comment by jeromelaban** [comment]
We haven't tested this scenario at this time, `iframe` are very tricky as they have their own security concerns.

**Comment by DierkDroth** [comment]
@jeromelaban would you accept a PR which tries to not show a tooltip at all in that case - since it's actually not needed and rather confusing?

**Comment by jeromelaban** [comment]
Yes, if you're able to, though we would not be surprised if current page (outside the iframe) does not receive any event that can be detected.

**Comment by dr1rrb** [comment]
Looks like we no longer receive the pointer events once it's over the `iframe`.

Few tips:

The best option would to hide the tooltip. the same we do when the element does out of the bounds of the window. i.e. by calling [`RaiseRecoverableLeave`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/DragDrop/DragOperation.cs#L150).

About the windows tooltip : it's not supposed to appear at all thanks to the [`preventDefault()` here](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/ts/Windows/ApplicationModel/DataTransfer/DragAndDropExtension.ts#L139).

Also you are able to customize what appears in the tooltip using [`DragEventArgs.DragUIOverride.Is<Caption|Content|Glyph>Visible`](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.draguioverride?view=winrt-22621). Not sure of what window's behavior if you set all of them to `false`, but I suspect that with uno we could just hide the complete `DragView` [here](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/DragDrop/DragView.cs#L159).

**Comment by DierkDroth** [comment]
Thanks for you input, guys. I'll try to work around the issue for now by tweaking the `DargEventArgs.DragUIOverride` property.
